R/database.R

```diff
@@ -1903,92 +1921,58 @@ get_sample_cn_segments = function(this_sample_id,
     }
     # check permissions to ICGC data
     permissions = file.access(full_cnv_path, 4)
-    if (permissions == -1) {
+    if(permissions == -1){
       message("restricting to non-ICGC data")
       cnv_flatfile_template = check_config_value(config::get("results_flatfiles")$cnv_combined$gambl)
       cnv_path =  glue::glue(cnv_flatfile_template)
       full_cnv_path =  paste0(check_config_value(config::get("project_base")), cnv_path)
     }
-
+    
     #check for missingness
     if(!file.exists(full_cnv_path)){
       print(paste("missing: ", full_cnv_path))
       message("Cannot find file locally. If working remotely, perhaps you forgot to load your config (see below) or sync your files?")
       message('Sys.setenv(R_CONFIG_ACTIVE = "remote")')
     }
-
+    
+    #read seg file and subset to sample IDs of interest
     all_segs = suppressMessages(read_tsv(full_cnv_path))
-    if (!missing(this_sample_id) & !multiple_samples) {
```

Comment:
The SQL retreival of the results seems to be still supported in the new code below this deleted chunk, so not sure what are the changes here?

Reply:
@mattssca can you clarify?